### PR TITLE
MAGECLOUD-2395 Add health notifications to integrations

### PR DESCRIPTION
**Pull request by hguthrie**
Added Health notifications to the Integrations for Cloud.

- Updated the Integrations landing page to include more than Git-hosting
- Added the new Health notifications topic
- Moved Integrations further down the navigation so it appears after local development setup
- Initially, I thought this would go in the configure notifications sections, but I realized this was not the case. There are a few minor changes, such as fixing a reference to the sample env YAML file, which doesn't exist.

These changes affect:
https://devdocs.magento.com/guides/v2.2/cloud/integrations/cloud-integrations.html
https://devdocs.magento.com/guides/v2.2/cloud/integrations/health-notifications.html

whatsnew
Added [Health notifications](https://devdocs.magento.com/guides/v2.2/cloud/integrations/health-notifications.html) to [Integrations](https://devdocs.magento.com/guides/v2.2/cloud/integrations/cloud-integrations.html) for Cloud.

### CODE REVIEW

File: guides/v2.1/cloud/env/variables-intro.md [hunk 1]
```diff
@@ -64,7 +66,9 @@ return array(
    ...
 );
 ```
-{% include note.html type="info" content="JS bundling and JS/CSS merging do not work with SCD on demand." %}
+
+{: .bs-callout .bs-callout-info}
+JS bundling and JS/CSS merging do not work with SCD on demand.
```
Comment (mveeramneni) [inline guides/v2.1/cloud/env/variables-intro.md]: we might have to double check this warning

Comment (hguthrie) [reply guides/v2.1/cloud/env/variables-intro.md]: Yeah, I don't see it in the DIST packaged in ece-tools. Will remove.

File: guides/v2.1/cloud/integrations/health-notifications.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+#### To register health notifications in a Slack channel:
+
+```bash
+magento-cloud integration:add --type health.slack --token SLACK_BOT_TOKEN --channel '#slack-channel-name'
+```
+
+## PagerDuty notifications
+
+PagerDuty is an external service that can notify on-call team members of important issues. Before you can receive health notifications in PagerDuty, you must create a [PagerDuty integration](https://v2.developer.pagerduty.com/v2/docs/integrating) that uses the Events API version 2. Use the Integration Key as known as the _routing key_ to register your integration.
```
Comment (meker12) [inline guides/v2.1/cloud/integrations/health-notifications.md]: extra "as"

File: guides/v2.1/cloud/env/log-handlers.md [hunk 3]
```diff
@@ -47,3 +47,16 @@ log:
...
 ```
+
+### Log levels
+
+Log levels determine the level of detail your notification messages contain. You can choose from the following options:
+
+-   **debug**—Detailed debug information.
+-   **info**—Interesting events. For example, a user logs in, SQL logs, etc.
+-   **notice**—Normal but significant events.
+-   **warning**—Exceptional occurrences that are not errors. For example, use of deprecated APIs, poor use of an API, undesirable things that are not necessarily wrong.
```
[the remark this answers is not in the document]
Comment (hguthrie) [reply guides/v2.1/cloud/env/log-handlers.md]: It's so romantic; don't you think?

**Comment by hguthrie** [comment]
Done; thanks for the review.

**Pull request merged**